docs/using/administering/upgrading.rst

```diff
@@ -0,0 +1,40 @@
+.. _upgrading-flocker:
+
+=========
+Upgrading
+=========
+
+A Flocker cluster can be upgraded to a newer version of Flocker while preserving all data and configuration of the Flocker cluster provided certain steps are followed.
+The correct steps to follow may vary depending on the versions of Flocker being upgraded from and to.
+A common requirement for all currently supported upgrade paths is that all nodes and the control service must be running the same version of Flocker.
+
+Flocker 1.0.1
+-------------
+
+Recommended Steps
+^^^^^^^^^^^^^^^^^
+
+  #. Stop the control service and the agent services on all nodes.
+  #. Install Flocker 1.0.1 on all hosts in the Flocker cluster.
+  #. Reboot each of the agent nodes.
+  #. If you have not configured the Flocker agents to start automatically on boot,
+     restart the agent services on all nodes.
+  #. Restart the control service.
```

Comment:
Too much indentation here; results in the list items being block quoted too.
 * http://build.clusterhq.com/results/docs/release-maintenance/flocker-1.0.0/blockdevicedeployer-upgrade-old-filesystems-FLOC-2376/build-8328/using/administering/upgrading.html#recommended-steps